service/src/main/java/de/adorsys/psd2/sandbox/SandboxApplication.java

```diff
@@ -51,9 +46,19 @@ public static void main(String[] args) {
           .run(args);
     } else {
       new SpringApplicationBuilder()
-          .parent(SandboxApplication.class).web(false)
-          .child(PortalConfig.class).listeners(new StartFailedListener()).web(true)
-          .sibling(Xs2aConfig.class).listeners(new StartFailedListener()).web(true)
+          .parent(SandboxApplication.class)
+          .web(false)
+          .initializers(new CustomNameConfigFileInitializer("sandbox-application"))
+
+          .child(PortalConfig.class)
+          .web(true)
+          .listeners(new StartFailedListener())
+          .initializers(new CustomNameConfigFileInitializer("portal-application"))
+
+          .sibling(Xs2aConfig.class)
+          .listeners(new StartFailedListener()).web(true)
```

Comment:
Minor format issue: move `.web(true)` into own line above, so every block is structured the same 
-> parent/child/sibling
.web
.listener (if needed)
.initializers